### PR TITLE
PD-561: Logic to determine when to display Realm/Stitch

### PR DESCRIPTION
**Pull request by brookescarlett**
## ✍️ Proposed changes

- Hard coding into the component to switch from Stitch to Realm

🎟 _Jira ticket:_ [PD-561](https://jira.mongodb.org/browse/PD-561)

## 🛠 Types of changes

- [ ] Tooling (updates to workspace config or internal tooling)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## ✅ Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [x] I have run `yarn changeset` and documented my changes

### CODE REVIEW

File: packages/mongo-nav/src/project-nav/ProjectNav.tsx [hunk 1]
```diff
@@ -198,6 +198,17 @@ const tooltipStyles = css`
   text-align: center;
 `;
 
+function displayProductName() {
```
Comment (hswolff) [inline packages/mongo-nav/src/project-nav/ProjectNav.tsx]: I'd just do this one and save it in a variable. No need to keep running this function on every render.

Comment (brookescarlett) [reply packages/mongo-nav/src/project-nav/ProjectNav.tsx]: secondTabName it is

File: packages/mongo-nav/src/project-nav/ProjectNav.tsx [hunk 2]
```diff
@@ -229,6 +240,8 @@ export default function ProjectNav({
   const isCloudManager = current?.planType === PlanType.Cloud;
   const onElementClick = useOnElementClick();
 
+  const secondTabName = displayProductName();
```
Comment (hswolff) [inline packages/mongo-nav/src/project-nav/ProjectNav.tsx]: You can make this variable outside of the component!

Comment (brookescarlett) [reply packages/mongo-nav/src/project-nav/ProjectNav.tsx]: Sorry duh that was the whole point

File: packages/mongo-nav/src/project-nav/ProjectNav.spec.tsx [hunk 3]
```diff
@@ -169,4 +171,51 @@ describe('packages/mongo-nav/src/project-nav', () => {
 
     testForAlerts(alerts, true);
   });
+
+  describe('when the date is before MongoDB World', () => {
+    test('Stitch is displayed in the ProjectNav', () => {
+      const productName = displayProductName();
```
Comment (hswolff) [inline packages/mongo-nav/src/project-nav/ProjectNav.spec.tsx]: You should mock this, cuz after May 4th this test will fail.

File: packages/mongo-nav/src/project-nav/ProjectNav.tsx [hunk 4]
```diff
@@ -198,6 +198,19 @@ const tooltipStyles = css`
   text-align: center;
 `;
 
+export function displayProductName() {
```
Comment (hswolff) [inline packages/mongo-nav/src/project-nav/ProjectNav.tsx]: FYI another strategy for testing the dates is to do this:

```suggestion
export function displayProductName(today = new Date(Date.now()), ) {
```

File: packages/mongo-nav/src/project-nav/ProjectNav.spec.tsx (hunk 3 above)
Comment (brookescarlett) [reply packages/mongo-nav/src/project-nav/ProjectNav.spec.tsx]: Smart 👍

File: packages/mongo-nav/src/project-nav/ProjectNav.tsx (hunk 4 above)
Comment (brookescarlett) [reply packages/mongo-nav/src/project-nav/ProjectNav.tsx]: Even better...

**Pull request merged**